runtime/vm/ContinuationHelpers.cpp

```diff
@@ -151,29 +111,46 @@ enterContinuation(J9VMThread *currentThread, j9object_t continuationObject)
 
 	if (started) {
 		/* resuming Continuation from yield */
-		resumeContinuation(currentThread, continuation);
+		VM_OutOfLineINL_Helpers::restoreInternalNativeStackFrame(currentThread);
+		VM_OutOfLineINL_Helpers::returnSingle(currentThread, JNI_TRUE, 1);
+		result = FALSE;
 	} else {
-		/* start new */
-		UDATA args[] = { (UDATA) continuationObject };
-		J9NameAndSignature executeNameAndSig = { (J9UTF8*)&execute_name, (J9UTF8*)&execute_sig };
-
+		/* start new Continuation execution */
 		J9VMJDKINTERNALVMCONTINUATION_SET_STARTED(currentThread, continuationObject, JNI_TRUE);
-
-		runStaticMethod(currentThread, J9UTF8_DATA(&continuationClass_name), &executeNameAndSig, 1, args);
+		/* prepare callin frame, send method will be set by interpreter */
+		J9SFJNICallInFrame *frame = ((J9SFJNICallInFrame*)currentThread->sp) - 1;
+		UDATA flags = J9_SSF_REL_VM_ACC;
```

Comment:
I dont think you need any of the off-load stuff, since we are remaining in the interpreter with this approach